osu.Game/Overlays/Profile/RankGraph.cs

```diff
@@ -124,9 +124,9 @@ private void userChanged(User user)
 
         private void updateRankTexts()
         {
-            rankText.Text = User.Value.Statistics.Rank > 0 ? $"#{User.Value.Statistics.Rank:#,0}" : "no rank";
+            rankText.Text = User.Value.Statistics.Ranks.Global > 0 ? $"#{User.Value.Statistics.Ranks.Global:#,0}" : "no rank";
             performanceText.Text = User.Value.Statistics.PP != null ? $"{User.Value.Statistics.PP:#,0}pp" : string.Empty;
-            relativeText.Text = $"{User.Value.Country?.FullName} #{User.Value.CountryRank:#,0}";
+            relativeText.Text = $"{User.Value.Country?.FullName} #{User.Value.Statistics.Ranks.Country:#,0}";

```

Comment:
Can country rank also be zero? I'd assume that could be the case if there's a check for the global one displaying "~~unranked~~ no rank" instead.

Reply:
nice catch